src/main/pages/user-management/user-management.adoc

```diff
@@ -59,16 +87,16 @@ Setting it to `embedded` will make it use Che’s OAuth Authenticator, for which
 Read more about link:ide_projects.html#project-import-and-ssh-connection[SHH key management].
 
 [id="protocol-based-providers"]
-== Protocol Based Providers
+== Protocol based providers
 
-Keycloak provides support for http://www.Keycloak.org/docs/3.2/server_admin/topics/identity-broker/saml.html[SAML v2.0] and http://www.Keycloak.org/docs/3.2/server_admin/topics/identity-broker/oidc.html[OpenID Connect v1.0] protocols so you can connect your identity provider systems if they support these protocols.
+Keycloak supports http://www.Keycloak.org/docs/3.2/server_admin/topics/identity-broker/saml.html[SAML v2.0] and http://www.Keycloak.org/docs/3.2/server_admin/topics/identity-broker/oidc.html[OpenID Connect v1.0] protocols. You can connect your identity provider systems if they support these protocols.
 
 [id="managing-users"]
-== Managing Users
```

Comment:
Why not use the orig. action title?

Reply:
since the content did not have a list of step or a procedure..I changed it to a concept heading.. But I see now..